### ISSUE TITLE
[BUG] Internal Command

### ISSUE THREAD

**Issue by ZeroRaven23** [Jules-WinnfieldX/CyberDropDownloader#122]
I'm trying to download and whenever I try to run it says that
**"'cyberdrop-dl' is not recognized as an internal command
or external, an operable program or a batch file."**
I already reinstalled python, and still it continues

Windowns 11

**Comment by Jules-WinnfieldX** [comment]
This is most often caused by python not being included on path. Make sure you ticked the box for 'add python to path' during installation.

If that didn't work, follow what another user did:
I don't know if this will help anyone having my issue, but after 2 reinstalls of python with path selected I was still receiving cyberdrop-dl not recognized error. So I ran python -m pip list -v , in cmd to see where cyperdrop was like was mentioned a few pages back. Then I went to AppData\Roaming\Python\Python310\site-packages and I removed the 2 cyperdrop folders that were in there. I then opened cmd and ran pip3 install cyberdrop-dl. Now the scraper works for me.
